builtin/providers/azurerm/resource_arm_managed_image_test.go

```diff
@@ -0,0 +1,345 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMManagedImage_standaloneImage(t *testing.T) {
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMManagedImage_standaloneImage, ri, ri)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMManagedImageDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMManagedImageExists("azurerm_managed_image.test", true),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMManagedImage_customImageVMFromVHD(t *testing.T) {
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMManagedImage_customImage_fromVHD, ri, ri, ri, ri)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMManagedImageDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMManagedImageExists("azurerm_managed_image.test", true),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMManagedImage_customImageVMFromVM(t *testing.T) {
+	ri := acctest.RandInt()
+	config := fmt.Sprintf(testAccAzureRMManagedImage_customImage_fromVM, ri, ri, ri, ri, ri)
```

Comment:
Looks like all Terraform tests currently use fmt.Sprintf in the tests - should we change that in our tests going forward?